README.md

```diff
@@ -111,39 +111,39 @@ Enable withExEditor after the upgrade.
 
-    #! /usr/bin/env bash
+ ```
+#! /usr/bin/env bash
 
-    # Install the host for withExEditor that allows editing text in Firefox using a text editor like Vim
-    # After executing this script, reload the Firefox plugin
+# Install the host for withExEditor that allows editing text in Firefox using a text editor like Vim.
+# After executing this script, reload the Firefox plugin.
 
-    # See https://github.com/asamuzaK/withExEditorHost/releases for supported operating systems
-    os="linux-x86_64"
-    # Allowed tags: "latest" and "next" (pre-release)
-    versionTag="latest"
-    # The host's version number
-    version=$(curl --silent https://registry.npmjs.org/withexeditorhost | jq --raw-output ".\"dist-tags\".\"$versionTag\"")
+# See https://github.com/asamuzaK/withExEditorHost/releases for supported operating systems
+os="linux-x86_64"
+# Allowed tags: "latest" and "next" (pre-release)
+versionTag="latest"
+# The host's version number
+version=$(curl --silent https://registry.npmjs.org/withexeditorhost | jq --raw-output ".\"dist-tags\".\"${versionTag}\"")
 
-    withExEditorHostRemoteFile="https://github.com/asamuzaK/withExEditorHost/releases/download/v"$version"/"$os".zip"
-    withExEditorHostLocalZipFile="/tmp/withExEditorHost.zip"
-    withExEditorHostDir="$HOME/.local/bin/withExEditorHost"
+withExEditorHostRemoteFile="https://github.com/asamuzaK/withExEditorHost/releases/download/v"${version}"/"${os}".zip"
+withExEditorHostLocalZipFile="/tmp/withExEditorHost.zip"
+withExEditorHostDir="$HOME/.local/bin/withExEditorHost"
 
-    echo "Downloading withExEditorHost "$version""
+echo "Downloading withExEditorHost "${version}""
 
-    # Create the dir for the host's index file, download the archive and unzip it
-    mkdir --parents "$withExEditorHostDir"
-    # If the URL returns 404, make cURL fail. This prevents 7z from unzipping an HTML error page
-    curl --fail -L -o "$withExEditorHostLocalZipFile" "$withExEditorHostRemoteFile"\
-    && 7z x "$withExEditorHostLocalZipFile" -o"$withExEditorHostDir"
+# Create the dir for the host's index file, download the archive and unzip it
+mkdir --parents "${withExEditorHostDir}"
+# If the URL returns 404, make cURL fail. This prevents 7z from unzipping an HTML error page
+curl --fail -L -o "${withExEditorHostLocalZipFile}" "${withExEditorHostRemoteFile}"\
+&& 7z x "${withExEditorHostLocalZipFile}" -o"${withExEditorHostDir}"
 
-    indexFile="$withExEditorHostDir/index"
-    hostScript="$HOME/.config/withexeditorhost/config/firefox/withexeditorhost.sh"
-
-    # The Firefox plugin will use this shell script to call the host's index file
-    printf "#! /usr/bin/env bash\n'$indexFile'\n" > "$hostScript"
-
-    chmod +x "$indexFile" "$hostScript"
+indexFile="${withExEditorHostDir}/index"
+hostScript=""${HOME}"/.config/withexeditorhost/config/firefox/withexeditorhost.sh"
```

Comment:
Just a question.
Are these quotes around `"${HOME}"` needed?

Reply:
I added the quotes in case `$HOME` contains spaces. I'm not sure if that's allowed in Linux.

Reply:
There are quotes at the beginning and the end, so `"${HOME}/.config/withexeditorhost/config/firefox/withexeditorhost.sh"` should be treated as a single string, I think.

Reply:
To me, current implement seems to be parsed as:
[empty string] [unquoted ${HOME}] [quoted .config/...]

Reply:
Makes sense. I removed those quotes.